docs/representing_statistics.md

```diff
@@ -0,0 +1,127 @@
+# Representing Statistics in Data Commons
+
+> Note: this document assumes familiarity with the
+[Schema.org Data Model](https://schema.org/docs/datamodel.html) and
+[MCF format](mcf_format.md).
+
+Data Commons adopts the
+[Schema.org Data Model](https://schema.org/docs/datamodel.html),
+and its schema is a superset of
+[Schema.org schema](https://schema.org/docs/schemas.html).
+
+The most commonly used high-level schema types for representing statistical data are:
+
+- [`Class`](https://schema.org/Class)
+- [`Property`](https://schema.org/Property)
+- [`StatisticalPopulation`](https://schema.org/StatisticalPopulation)
+- [`Observation`](https://schema.org/`Observation`)
+
+- [`Curator`](https://browser.datacommons.org/kg?dcid=Curator)
+- [`Provenance`](https://browser.datacommons.org/kg?dcid=Provenance)
+- [`StatisticalVariable`](https://browser.datacommons.org/kg?dcid=StatisticalVariable)
+- [`StatVarObservation`](https://browser.datacommons.org/kg?dcid=StatVarObservation)
+
+The first four are defined in Schema.org, and the latter four are Data Commons
+extensions.
+
+`Curator` and `Provenance` are necessary for representing who published
+the data to Data Commons and where the data is sourced from.
+`StatisticalVariable` and `StatVarObservation` are relatively new types,
+introduced to reduce the data import and usage complexities of
+`StatisticalPopulation` and `Observation` schema.
+
+## Intro to `StatisticalVariable` and `StatVarObservation`
+
+[`StatisticalVariable`](https://browser.datacommons.org/kg?dcid=StatisticalVariable)
+represents any type of statistical metric that can be measured at a place and
+time. Some examples include: median income, median income of females,
+number of high school graduates, unemployment rate,
+prevalence of diabetes, essentially anything you might call a metric,
+statistic, or measure.
+
+[`StatVarObservation`](https://browser.datacommons.org/kg?dcid=StatVarObservation)
+represents an actual measurement of a `StatisticalVariable` in a given place and time.
+
+The statement "According to the US Census ACS 5 Year Estimates,
+the median age of people in San Antonio, Texas in 2014 was 39.4."
+can be represented as:
+
+```
+Node: Observation_MedianAge_SanAntonio_TX_2014
+typeOf: dcs:StatVarObservation
+variableMeasured: dcid:MedianAge
+observationAbout: dcid:geoId/4865000
+observationDate: "2014"
+value: 39.4
+```
+
+Where [`MedianAge`](https://browser.datacommons.org/kg?dcid=MedianAge)
+is a `StatisticalVariable` schema node that only needs to be defined once.
+
+
+## Mapping `StatisticalVariable` and `StatVarObservation` to `StatisticalPopulation` and `Observation`
```

Comment:
The content here sort presupposes user understands StatPop/Obs. Should we link in the data-model doc that delves into those? Like the one that I think Antares curated.

Reply:
I will do this and the test in a separate PR.